requirements.txt

```diff
@@ -9,62 +13,79 @@ certifi==2020.11.8
 cffi==1.14.4
 chardet==3.0.4
 click==7.1.2
+coverage==5.3
 cryptography==3.2.1
 dash==1.17.0
 dash-bootstrap-components==0.10.7
 dash-core-components==1.13.0
 dash-html-components==1.1.1
 dash-renderer==1.8.3
 dash-table==4.11.0
+dateparser==1.0.0
+decorator==4.4.2
 dnspython==2.0.0
 email-validator==1.1.2
 Flask==1.1.2
+Flask-Admin==1.5.7
 Flask-Bcrypt==0.7.1
 Flask-Compress==1.8.0
 Flask-Login==0.5.0
 Flask-Mail==0.9.1
+Flask-Migrate==2.5.3
 Flask-SQLAlchemy==2.4.4
 Flask-WTF==0.14.3
 future==0.18.2
 idna==2.10
 iniconfig==1.1.1
+ipython==7.19.0
+ipython-genutils==0.2.0
 isort==5.6.4
 itsdangerous==1.1.0
+jedi==0.17.2
 Jinja2==2.11.2
 jmespath==0.10.0
 lazy-object-proxy==1.4.3
+Mako==1.1.3
 MarkupSafe==1.1.1
 mccabe==0.6.1
+moment==0.12.1
 numpy==1.19.4
 oauthlib==3.1.0
 packaging==20.4
 pandas==1.1.4
+parso==0.7.1
+pexpect==4.8.0
+pickleshare==0.7.5
 Pillow==8.0.1
 plotly==4.13.0
 pluggy==0.13.1
+prompt-toolkit==3.0.8
+ptyprocess==0.6.0
 py==1.9.0
 pycodestyle==2.6.0
 pycparser==2.20
+Pygments==2.7.3
 pylint==2.6.0
 PyMySQL==0.10.1
 pyparsing==2.4.7
 pytest==6.1.2
 python-dateutil==2.8.1
+python-editor==1.0.4
 pytz==2020.4
+regex==2020.11.13
 requests==2.25.0
 retrying==1.3.3
 s3transfer==0.3.3
 six==1.15.0
 SQLAlchemy==1.3.20
+timeago==1.0.14
+times==0.7
 toml==0.10.2
 tqdm==4.53.0
+traitlets==5.0.5
+tzlocal==2.1
 urllib3==1.26.2
+wcwidth==0.2.5
 Werkzeug==1.0.1
 wrapt==1.12.1
 WTForms==2.3.3
-Flask-Admin==1.5.7
-Flask-migrate==2.5.3
```

Comment:
Is it OK to remove the flask-admin package?

Reply:
it is moved, not removed.

Reply:
cool